docs-ref-conceptual/index.yml

```diff
@@ -57,10 +57,10 @@ sections:
     className: cardsM
     columns: 3
     items: 
-    - href: https://docs.microsoft.com/azure/app-service/containers/quickstart-nodejs
-      title: Create a Node.js app
+    - href: https://code.visualstudio.com/docs/nodejs/nodejs-deployment
```

Comment:
Can you include the GA tracking parameters in this? `?utm_source=msdocs&utm_medium=node_hub`

Reply:
sure!